addons/website/static/src/snippets/s_floating_blocks/options.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="website.s_floating_blocks.alert.empty">
```

Comment:
Don't indent the first level of XML files in website (add blank line around each template though)